vpn_slice/__main__.py

```diff
@@ -109,6 +114,22 @@ def names_for(host, domains, short=True, long=True):
         elif rest in domains: names.append(first)
     return names
 
+def read_routes_file(filename):
+    hosts = []
+    with open(filename) as f:
+        lines = [x.strip() for x in f.readlines()]
+        for line in lines:
```

Comment:
No need to read the entire file in memory beforehand, you can just iterate over the lines like this:

```suggestion
        for line in f:
            line = line.strip()
```